tests/config/test_schema.py

```diff
@@ -4,50 +4,27 @@
 import unittest
 
 from importlib_metadata import entry_points
+import pydantic
+from pydantic import TypeAdapter
 
 from bugwarrior.config import schema
 
 from ..base import ConfigTest
 
 
-class TestLoggingPath(unittest.TestCase):
+class TestConfigList(unittest.TestCase):
```

Comment:
Seems like we were indirectly testing ExpandedPath here.

```suggestion
class TestExpandedPath(unittest.TestCase):
    def setUp(self):
        self.adapter = TypeAdapter(schema.ExpandedPath)
        self.dir = os.getcwd()
        os.chdir(os.path.expanduser('~'))
        self.log = pathlib.Path('./bugwarrior.log').absolute()

    def test_log(self):
        filename = os.path.join(os.path.expandvars('$HOME'), self.log)
        self.assertEqual(self.adapter.validate_python(filename), self.log)

    def test_log_userhome(self):
        self.assertEqual(self.adapter.validate_python('~/bugwarrior.log'), self.log)

    def test_log_envvar(self):
        self.assertEqual(self.adapter.validate_python('$HOME/bugwarrior.log'), self.log)

    def tearDown(self):
        os.chdir(self.dir)


class TestConfigList(unittest.TestCase):
```